include/PianoRoll.h

```diff
@@ -125,6 +125,7 @@ class PianoRoll : public QWidget
 	virtual void focusOutEvent( QFocusEvent * );
 
 	int getKey( int y ) const;
+	QList<int> getAllOctavesForKey( int keyToMirror ) const;
```

Comment:
@Wallacoloo I suppose it could be, I had included it with getKey() under the protected access, in order to be able to use them in similar contexts if subclasses were made, but it  probably makes sense to put it under private access, as I doubt there would be much need for getting the octave-periodic keys outside the context of this specific functionality.